server/neptune/workflows/internal/activities/audit.go

```diff
@@ -0,0 +1,57 @@
+package activities
+
+import (
+	"context"
+	"strconv"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/deploy/request"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/job"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/root"
+)
+
+type writer interface {
+	Write([]byte) error
+}
+
+type auditActivities struct {
+	SnsWriter writer
+}
+
+type AuditJobRequest struct {
+	DeploymentInfo root.DeploymentInfo
+	State          job.State
+}
+
+func (a *auditActivities) AuditJob(ctx context.Context, req AuditJobRequest) error {
+	isForceApply := req.DeploymentInfo.Root.Trigger == root.Trigger(request.ManualTrigger)
+	atlantisJobEvent := &job.Event{
+		Version:        1,
+		ID:             req.DeploymentInfo.ID,
+		RootName:       req.DeploymentInfo.Root.Name,
+		JobType:        job.ApplyJob,
+		Repository:     req.DeploymentInfo.Repo.GetFullName(),
+		Environment:    ctx.Value("environment").(string),
+		InitiatingUser: req.DeploymentInfo.User.Username,
+		Project:        ctx.Value("service_name").(string),
```

Comment:
When is `environment` and `service_name` set in the context?

Reply:
I was originally planning to set this value in the context but decided to pass in tags similar to our project context since these fields are specific to lyft's use case.